test/SSODataTest.php

```diff
@@ -26,18 +26,25 @@ class SSODataTest extends TestCase
      * Can be used in development in conjunction with
      * createSignedTokenFromData to issue development tokens.
      *
+     * @param string|null $exp
+     * @param string|null $npf
+     * @param string|null $iat
      * @return array Associative array of claims.
+     * @throws \Exception
      */
-    public static function getTokenData()
+    public static function getTokenData(?string $exp = '10 minutes', ?string $npf = '-1 minutes', ?string $iat = 'now')
     {
+        $exp = $exp ?? '10 minutes';
+        $npf = $npf ?? '-1 minutes';
+        $iat = $iat ?? 'now';
```

Comment:
I don't like these parameter names. Please consider to change into something more meaningful but feel free to not change them if they follow a common pattern.

Reply:
It's the test file and this are the claim namings, so I would keep it.